client/web/web.go

```diff
@@ -303,8 +304,13 @@ var (
 //
 //   - (errNoSession) The request does not have a session.
 //
-//   - (errTaggedSource) The source is a tagged node. Users must use their
-//     own user-owned devices to manage other nodes' web clients.
+//   - (errTaggedRemoteSource) The source is remote (another node) and tagged.
+//     Users must use their own user-owned devices to manage other nodes'
+//     web clients.
+//
+//   - (errTaggedLocalSource) The source is local (the same node) and tagged.
+//     Tagged nodes can only be remotely managed, allowing ACLs to dictate
```

Comment:
I remember us talking about this a lot, but honestly can't remember where we landed.  I think it was that we should be able to allow this eventually, but we'd need to add logic on control to very that the check-mode user is an owner of the tags?  Or something like that?  I guess I do recall that it's an error state.

Reply:
Hahah yea we went back and forth on it a lot. I think we really could figure out something that works locally (using tag owners), but didn't quite settle on anything yet, so went for requiring remote management for now. Can always add it once we think that scenario through some more!